commit ca0e10950a5db60121ac741f0bad28af9bb504b9
Author: Natanael Copa <ncopa@alpinelinux.org>
Date:   Tue Dec 28 13:27:55 2010 +0000

    setup-interfaces: use vi as default editor
    
    since nano is not shipped with mini iso.

diff --git a/setup-interfaces.in b/setup-interfaces.in
index aad042d..b96ee58 100755
--- a/setup-interfaces.in
+++ b/setup-interfaces.in
@@ -141,12 +141,11 @@ while [ "$answer" != "yes" ] && [ "$answer" != "no" ] ; do
 done
 
 if [ "$answer" = "yes" ]; then
-	[ -z "$EDITOR" ] && EDITOR=nano
-	case $EDITOR in
+	case "$EDITOR" in
 		nano)	pkg_inst nano;;
 		vim)	pkg_inst vim;;
 	esac
-	$EDITOR interfaces
+	${EDITOR:-vi} interfaces
 fi
 
 mkdir -p $ROOT/etc/network
